docs/reference/query-dsl/geo-polygon-query.asciidoc

```diff
@@ -34,8 +34,9 @@ points. Here is an example:
 |Option |Description
 |`_name` |Optional name field to identify the filter
 
-|`ignore_malformed` |Set to `true` to accept geo points with invalid latitude or
-longitude (default is `false`).
+|`validation_method` |Set to `IGNORE_MALFORMED` to accept geo points with
+invalid latitude or longitude, `COERCE` to try and infer correct latitude
+or longitude, or `STRICT` (default is `STRICT`).
```

Comment:
Should we add deprecation warnings in the asciidoc to indicate that `ignore_malformed` and `coerce` are replaced by `validation_method`?